### PR TITLE
chore: Release/0.6.1

### PR DESCRIPTION
**Pull request by jrconlin**
#### Features
* update to actix-web 3 (#834)
* added more detailed release process to README.md

#### Bug Fixes
* return correct error code and value for OverQuota users (#837)

### CODE REVIEW

**Review by pjenvey — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -210,13 +210,27 @@ Functional tests live in [server-syncstorage](https://github.com/mozilla-service
...
-
-Once your PR merges, then go ahead and create an official [GitHub release](https://github.com/mozilla-services/syncstorage-rs/releases).
+1. Switch to master branch of syncstorage-rs
+2. `git pull` to ensure that the local copy is up-to-date.
+3. `git pull origin master` to make sure that you've incorporated any changes to the master branch.
+4. `git diff origin/master` to ensure that there are no local staged or uncommited changes.
+5. Bump the version number in [Cargo.toml](https://github.com/mozilla-services/syncstorage-rs/blob/master/Cargo.toml) (this new version number will be designated as `<version>` in this checklist)
+6. create a git branch for the new version `git checkout -b release/<version>`
+7. `cargo build --release` - Build with the release profile [release mode](https://doc.rust-lang.org/book/ch14-01-release-profiles.html).
+8. `clog -C CHANGELOG.md` - Generate release notes. We're using [clog](https://github.com/clog-tool/clog-cli) for release notes. Add a `-p`, `-m` or `-M` flag to denote major/minor/patch version, ie `clog -C CHANGELOG.md -p`.
```
Comment (pjenvey) [inline README.md:220]: I do clog `--setversion <version>`, it's a little more "automatic" than clog's bumping flags (I didn't even know those existed)

File: README.md [hunk 2]
```diff
@@ -210,13 +210,27 @@ Functional tests live in [server-syncstorage](https://github.com/mozilla-service
...
-2. `cargo build --release` - Build with the release profile [release mode](https://doc.rust-lang.org/book/ch14-01-release-profiles.html).
-3. `clog -C CHANGELOG.md` - Generate release notes. We're using [clog](https://github.com/clog-tool/clog-cli) for release notes. Add a `-p`, `-m` or `-M` flag to denote major/minor/patch version, ie `clog -C CHANGELOG.md -p`.
-
-Once your PR merges, then go ahead and create an official [GitHub release](https://github.com/mozilla-services/syncstorage-rs/releases).
+1. Switch to master branch of syncstorage-rs
+2. `git pull` to ensure that the local copy is up-to-date.
+3. `git pull origin master` to make sure that you've incorporated any changes to the master branch.
+4. `git diff origin/master` to ensure that there are no local staged or uncommited changes.
+5. Bump the version number in [Cargo.toml](https://github.com/mozilla-services/syncstorage-rs/blob/master/Cargo.toml) (this new version number will be designated as `<version>` in this checklist)
+6. create a git branch for the new version `git checkout -b release/<version>`
```
Comment (pjenvey) [inline README.md:218]: I prefer `release/<major>.<minor>` but I can't recall if this matters much in practice

**Review by pjenvey — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jrconlin) [reply README.md:220]: Yeah, FWIW, I do it manually as well.

File: README.md (hunk 2 above)
Comment (jrconlin) [reply README.md:218]: Ah, I kinda rolled `<major>.<minor>.<patch>` into `<release>`, but I should make that clear.

**Pull request merged**